lnwallet/channel.go

```diff
@@ -742,13 +756,27 @@ func (lc *LightningChannel) diskCommitToMemCommit(
 		)
 	}
 
+	auxLeaves, err := AuxLeavesFromCommit(
+		lc.channelState, *diskCommit, lc.leafStore,
+		func() CommitmentKeyRing {
+			if whoseCommit.IsLocal() {
+				return *localCommitKeys
+			}
+
+			return *remoteCommitKeys
```

Comment:
If you pack local and remote commit keys into a dual, this just becomes `commitKeys.GetParty(whoseCommit)`

Reply:
Added a refactor commit.